imports/plugins/core/checkout/client/components/completedOrder.js

```diff
@@ -27,8 +28,15 @@ const CompletedOrder = ({ order, orderId, shops, orderSummary, paymentMethods, h
       />
     );
   }
-  return (
-    <div className="container order-completed">
+
+  let headerText;
```

Comment:
This component should not need to be aware of what route it is in. It should be handled by passing in a prop. Components should be as dumb as possible.